tests/test_visitors/test_ast/test_general/test_wrong_name/test_variable_names.py

```diff
@@ -162,3 +163,23 @@ def test_correct_variable_name(
     visitor.run()
 
     assert_errors(visitor, [])
+
+
+@pytest.mark.parametrize('code', [
```

Comment:
Please add happy path tests. We should test that variables like `test34` and `test32_109` are fine.

Reply:
Sure